internal/pkg/egressacl/yaml.go

```diff
@@ -0,0 +1,132 @@
+package egressacl
+
+import (
+	"errors"
+	"fmt"
+	_ "github.com/stripe/smokescreen/pkg/egressacl"
+	"github.com/stripe/smokescreen/pkg/egressacl/decision"
+	"github.com/stripe/smokescreen/pkg/egressacl/enforcementpolicy"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"log"
+	"regexp"
+)
+
+type EgressAclYamlEntry struct {
```

Comment:
The class naming is kind of weird here-- `EgressAclConfiguration` maps directly to the YAML configuration file, but `EgressAclYaml` and `EgressAclYamlEntry` are the internal config format that doesn't map directly to YAML.

Reply:
Yeah, I was a bit lenient with this one since it's just an implementation detail behind an interface. I hear you though!